contracts/SurplusBeneficiary.sol

```diff
@@ -0,0 +1,122 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity 0.7.6;
+pragma abicoder v2;
+
+import { Address } from "@openzeppelin/contracts/utils/Address.sol";
+import { SafeMath } from "@openzeppelin/contracts/math/SafeMath.sol";
+import { ReentrancyGuard } from "@openzeppelin/contracts/utils/ReentrancyGuard.sol";
+import { SafeERC20, IERC20 } from "@openzeppelin/contracts/token/ERC20/SafeERC20.sol";
+import { Ownable } from "@openzeppelin/contracts/access/Ownable.sol";
+import { SafeMath } from "@openzeppelin/contracts/math/SafeMath.sol";
+import { FullMath } from "@uniswap/v3-core/contracts/libraries/FullMath.sol";
+import { SurplusBeneficiaryStorageV1 } from "./storage/SurplusBeneficiaryStorage.sol";
+import { ISurplusBeneficiary } from "./interface/ISurplusBeneficiary.sol";
+import { IFeeDistributor } from "./interface/IFeeDistributor.sol";
+
+contract SurplusBeneficiary is ISurplusBeneficiary, ReentrancyGuard, Ownable, SurplusBeneficiaryStorageV1 {
```

Comment:
maybe no need to separate storage contract if it's immutable

Reply:
doh! 🤣